.bundle/config

```diff
@@ -0,0 +1,3 @@
+---
+BUNDLE_PATH: "./vendor"
+BUNDLE_DISABLE_SHARED_GEMS: '1'
```

Comment:
yeah this was for my setup. I prefer to store project specific gems in ./vendor gem dir.